steely/plugins/sort.py

```diff
@@ -0,0 +1,15 @@
+'''message -> aeegmss'''
+
+__author__ = 'iandioch'
+COMMAND = '.sort'
+
+
+def sort_message(message):
+    if not message:
+        return ""
+    out = ''.join(sorted(message))
+    return out
```

Comment:
`''.join(sorted(''))` works fine my dude

Reply:
it's all about efficiency. When this is running in 5 DCs with a few thou QPS we're going to be costing A Aron a few mil across the year trying to sort brandon's hollow empty messages.